### PR TITLE
CIV-10540 Cancel Trial Arrangements invalidates redis info

### PR DESCRIPTION
**Pull request by miguelMolina3691**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-10540

### Change description ###
Now, Cancel action in Trial Arrangement pages invalidates the data that is stored in Redis Data Cache Store

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/test/unit/routes/features/caseProgression/trialArrangements/cancelTrialArrangements.ts
Comment (LeonardoPalmeiro) [inline src/test/unit/routes/features/caseProgression/trialArrangements/cancelTrialArrangements.ts:1]: the name should be cancelTrialArrangementsController.test.ts if we don't add test.ts at the end we will probably have some problem

File: src/main/views/features/caseProgression/trialArrangements/has-anything-changed.njk [hunk 1]
```diff
@@ -60,7 +60,7 @@
               {{ govukButton({
                 text: t('COMMON.BUTTONS.CONTINUE')
               }) }}
-              <a href={{ latestUpdatesUrl }} class="govuk-link govuk-!-margin-left-2" rel="noreferrer noopener" >{{ t('COMMON.BUTTONS.CANCEL') }}</a>
+              <a href="{{ cancelUrl }}" class="govuk-link govuk-!-margin-left-2" rel="noreferrer noopener" >{{ t('COMMON.BUTTONS.CANCEL') }}</a>
```
Comment (LeonardoPalmeiro) [inline src/main/views/features/caseProgression/trialArrangements/has-anything-changed.njk:63]: in src/main/views/features/caseProgression/trialArrangements/hearing-duration-other-info.njk you added this part without **_""_** which one is the correct?

File: src/test/unit/routes/features/caseProgression/trialArrangements/cancelTrialArrangements.ts
Comment (miguelMolina3691) [reply src/test/unit/routes/features/caseProgression/trialArrangements/cancelTrialArrangements.ts:1]: Sure, I will modify it

File: src/main/views/features/caseProgression/trialArrangements/has-anything-changed.njk (hunk 1 above)
Comment (miguelMolina3691) [reply src/main/views/features/caseProgression/trialArrangements/has-anything-changed.njk:63]: I believe both are accepted although without double quotes webstorm complains. I modify hearing-duration-other-info to leave them all the same

**Review by LeonardoPalmeiro — APPROVED** [review]

**Pull request merged**